templates/step_function/consignment_export_definition.json.tpl

```diff
@@ -1,19 +1,136 @@
 {
   "Comment": "A state machine to run the Fargate task to export the consignment",
```

Comment:
Can you update the comment to reflect that the state machine now generates two types of export output.